hack/ci/windows.ps1

```diff
@@ -836,12 +836,44 @@ Try {
         }
     }
 
+    # Create loopback adapter for swarm integration tests if does not exist
+    if (($null -eq $env:LCOW_MODE) -and ($null -eq $env:LCOW_BASIC_MODE)) {
+        if (!(Get-NetAdapter -Name "eth0" -ErrorAction:SilentlyContinue)) {
+            $currentPATH = $env:PATH
+            $currentTEMP = $env:TEMP
+            Write-Host -ForegroundColor Green "INFO: Creating loopback adapter eth0"
+            Install-Module -Name LoopbackAdapter -Force
+            Import-Module -Name LoopbackAdapter
+            $nic = New-LoopbackAdapter -Name "eth0" -Force
+            Set-NetIPInterface -InterfaceIndex $nic.ifIndex -InterfaceMetric "254" -WeakHostReceive Enabled -WeakHostSend Enabled -DHCP Disabled
+            New-NetIPAddress -InterfaceIndex $nic.ifIndex -IPAddress "192.168.255.87" -PrefixLength 24
+            Disable-NetAdapterBinding -Name "eth0" -ComponentID ms_tcpip6 -PassThru
+
+            # LoopbackAdapter module overrides PATH abd TENO variabls so restore them back
```

Comment:
(I know it's still WIP, but noticed typos, so let me leave a comment in case I forget) 😅 

```suggestion
            # LoopbackAdapter module overrides PATH and TEMP variables so restore them back
```

Reply:
Have been WIP soon two years 😄 But anyway those configs most probably need to be replaced with some other solution anyway as it looks that even including loopback NIC does not stop HNS from reconfiguring networks during swarm init and leave...